lib/path.js

```diff
@@ -165,15 +164,9 @@ function glob(path, pattern, windows) {
   emitExperimentalWarning('glob');
   validateString(path, 'path');
   validateString(pattern, 'pattern');
-  return lazyMinimatch().minimatch(path, pattern, {
-    __proto__: null,
+  return lazyMicromatch().isMatch(path, pattern, {
     nocase: isWindows || isMacOS,
-    windowsPathsNoEscape: true,
     nonegate: true,
-    nocomment: true,
-    optimizationLevel: 2,
-    platform: windows ? 'win32' : 'posix',
-    nocaseMagicOnly: true,
```

Comment:
Is this feature equivalent with `minimatch`?

Reply:
> Is this feature equivalent with `minimatch`?

I’m looking into that now, once I get the test suite running… At least some of those options aren’t required/don’t exist.

Reply:
The options I'm concerned about is `windowsPathsNoEscape` and `platform`.

Reply:
Also,
`__proto__: null` should not be removed.

Reply:
> The options I'm concerned about is `windowsPathsNoEscape` and `platform`.

According to the [minimatch README](https://github.com/isaacs/minimatch#platform):

> platform

> Defaults to the value of `process.platform`.

So this option should not matter.

---

Quoting from the [micromatch README](https://github.com/micromatch/micromatch#contributing):

> Windows support - More reliable windows support than minimatch and multimatch.

> Platform issues

> It's important to us that micromatch work consistently on all platforms. If you encounter any platform-specific matching or path related issues, please let us know (pull requests are also greatly appreciated).

Reply:
We don't want it to default, we want it based on the input `windows` argument of the function.

Reply:
> Also, `__proto__: null` should not be removed.

Reverted.

> > Defaults to the value of `process.platform`.
> 
> We don't want it to default, we want it based on the input `windows` argument of the function. It was explicity set for a reason, but I don't know if `micromatch` will support an explicit value.

Well here is what it says [about backslashes](https://github.com/micromatch/micromatch#backslashes), there is also  the [`posixSlashes`](https://github.com/micromatch/micromatch#optionsposixslashes) option, with which I can do something like:
~~~ js
posixSlashes: !windows,
~~~

@RedYetiDev What _exactly_ is the expected difference in behaviour of passing in the `windows` boolean?

Care to chime in on the specifics here @jonschlinkert?

Reply:
I'm not *quite* sure, but @isaacs (who made `minimatch`) should know.